### DISCUSSION TITLE
Post processing

### DISCUSSION THREAD

**Discussion by vikashg** [ENHANCE-PET/MOOSE#103] [Q&A]
Hi all, 
I have a quick question regarding the PET-CT alignment. This question is regarding Moose v1.0. In 1.0, the model produces an image called `MOOSE-unified-PET-CT-atlas.nii.gz`. If I understand correctly, this is the fused label map. However, I would love a CT-registered to PET and a PET registered to CT. I believe that we can do that without any additional work because in the `temp/labels` folder, there is an affine matrix which I believe is the transformation matrix. 

What software (ITK/ANTs) can be used to apply this transformation matrix to the image? The reason I ask this question is because the matrix is not in the usual ITK format.

**Comment by vikashg** [comment]
@LalithShiyam for visibility.

**Reply by LalithShiyam** [reply]
This badass tool: https://greedy.readthedocs.io/en/latest/

**Reply by vikashg** [reply]
Thanks @LalithShiyam you are awesome. I was going through the code to find out :)

**Reply by LalithShiyam** [reply]
Ha ha - all good! Enjoy. I will fix your request in MOOSEv2. You should have it next week :)! This week is just not good.
